metadata-ingestion/docs/sources/dataplex/dataplex_pre.md

```diff
@@ -0,0 +1,190 @@
+   }
+   ```
+
+3. To provide credentials to the source, you can either:
+
+   Set an environment variable:
+
+   ```sh
+   $ export GOOGLE_APPLICATION_CREDENTIALS="/path/to/keyfile.json"
+   ```
+
+   _or_
+
+   Set credential config in your source based on the credential json file. For example:
+
+   ```yml
+   credential:
+     project_id: "project-id-1234567"
+     private_key_id: "d0121d0000882411234e11166c6aaa23ed5d74e0"
+     private_key: "-----BEGIN PRIVATE KEY-----\nMIIyourkey\n-----END PRIVATE KEY-----\n"
+     client_email: "test@suppproject-id-1234567.iam.gserviceaccount.com"
+     client_id: "123456678890"
+   ```
+
+### Integration Details
+
+The Dataplex connector extracts metadata from Google Dataplex, including Projects, Lakes, Zones, Assets, and Entities in a given project and location.
+
+#### Concept Mapping
+
+This ingestion source maps the following Dataplex Concepts to DataHub Concepts:
+
+| Dataplex Concept | DataHub Concept                                                                      | Notes                                                                                                 |
+| :--------------- | :----------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------- |
+| Project          | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | GCP Project containing Dataplex resources                                                             |
+| Lake             | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | Data lake container (sub-container of Project)                                                        |
+| Zone             | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | Zone container under a Lake (RAW or CURATED type)                                                     |
+| Asset            | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | Asset container representing a BigQuery dataset or GCS bucket (sub-container of Zone)                 |
+| Entity           | [`Dataset`](https://docs.datahub.com/docs/generated/metamodel/entities/dataset)      | Discovered table or fileset (linked to Asset container). Schema metadata is extracted when available. |
+
+#### Hierarchy
+
+The connector creates a hierarchical container structure:
+
+```
+Project (Container)
+└── Lake (Container)
+    └── Zone (Container)
+        └── Asset (Container)
+            └── Entity (Dataset)
+```
+
+#### Sibling Relationships
+
+When `create_sibling_relationships` is enabled, the connector creates bidirectional sibling relationships between Dataplex entities and their source platform entities (e.g., BigQuery tables, GCS objects). This allows you to:
```

Comment:
> between Dataplex entities and their source platform entities (e.g., BigQuery tables, GCS objects)

I find this a little bit confusing, what about something like? between entities ingested by the Dataplex source ingestor and those ingested by the Bigquery one. Or something like that.

Reply:
no siblings anymore, so this does not apply. resolving.